spec/features/end_to_end/provider_led_postgrad_journey_spec.rb

```diff
@@ -3,7 +3,7 @@
 require "rails_helper"
 
 feature "provider-led (postgrad) end-to-end journey", type: :feature do
-  let(:itt_start_date) { 1.week.from_now }
+  let(:itt_start_date) { 1.month.from_now }
```

Comment:
This change ensures that the chosen start date matches with the course, as defined here: https://github.com/DFE-Digital/register-trainee-teachers/blob/main/spec/support/features/course_details_steps.rb#L41